activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessAdminRuntimeImpl.java

```diff
@@ -261,6 +269,16 @@ public void setVariables(SetProcessVariablesPayload setProcessVariablesPayload)
 
     }
 
+    @Override
+    public List<VariableInstance> variables(GetVariablesPayload getVariablesPayload) {
```

Comment:
not sure if a more specific method name will help. Here we are fetching variables byProcessInstanceId why don't we pass only the processId string?

Reply:
it's same as ProcessRuntime https://github.com/Activiti/Activiti/blob/develop/activiti-api/activiti-api-process-runtime/src/main/java/org/activiti/api/process/runtime/ProcessRuntime.java#L134